users/faq.rst

```diff
@@ -494,3 +494,31 @@ to do this.  The most well known is called daemontools, and can be found in the
 standard package repositories for  almost every modern Linux distribution.
 Other popular tools with similar functionality include S6 and the aforementioned
 runit.
+
+.. _inotify-limits:
+
+How do I increase the inotify limit to get my fs watcher to work?
```

Comment:
I don't think "fs" is established enough to warrant a lower case abbreviation. `FS watcher" or "filesystem watcher"?